src/network/netdev/tunnel.c

```diff
@@ -745,7 +749,9 @@ static int netdev_tunnel_verify(NetDev *netdev, const char *filename) {
                 return log_netdev_error_errno(netdev, SYNTHETIC_ERRNO(EINVAL),
                                               "The local address cannot be '%s' when Independent= or AssignToLoopback= is enabled, ignoring.",
                                               strna(netdev_local_address_type_to_string(t->local_type)));
-
+        if (t->ignore_df && t->pmtudisc)
```

Comment:
How about to make `pmtudisc` tristate, then refuse to configure the tunnel when `DiscoverPathMTU=` is explicitly enabled?
```c
if (t->pmtudisc > 0 && t->ignore_df)
        return log_netdev_error_errno(...);
if (t->pmtudisc < 0)
        t->pmtudisc = !t->ignore_df;
```

Reply:
done. please have another look.